src/data_race.rs

```diff
@@ -120,10 +104,10 @@ pub enum AtomicFenceOp {
 /// of a thread, contains the happens-before clock and
 /// additional metadata to model atomic fence operations.
 #[derive(Clone, Default, Debug)]
-struct ThreadClockSet {
+pub struct ThreadClockSet {
     /// The increasing clock representing timestamps
     /// that happen-before this thread.
-    clock: VClock,
+    pub clock: VClock,
```

Comment:
Hm, we probably don't want *all* code to have access to this, do we? Just the weak_memory module? Maybe that should be put into a common submodule with this one.

Reply:
I made these `pub(crate)`